kuma/settings/testing.py

```diff
@@ -1,6 +1,7 @@
 from .local import *  # noqa
 
 DEBUG = False
+TEMPLATES[0]['OPTIONS']['debug'] = True  # Enable recording of templates
```

Comment:
I think, this should be depend on debug value. Moreover template debug should on in developer settings